config/BloodMagic/meteors/MASTER1_GemMaster.json

```diff
@@ -0,0 +1,61 @@
+{
+  "ores": [
+	"oreEndstoneDiamond",
+    "5",
+	"oreEndstoneEmerald",
+    "5",
+	"oreEndstoneRuby",
+    "5",
+	"oreEndstoneSapphire",
+    "5",
+	"oreEndstoneGreenSapphire",
+    "5",
+	"oreEndstoneOlivine",
+    "5",
+	"oreEndstoneTopaz",
+    "5",
+	"oreEndstoneTanzanite",
+    "5",
+	"oreEndstoneAmethyst",
+    "5",
+	"oreEndstoneOpal",
+    "5",
+	"oreEndstoneJasper",
+    "5",
+	"oreEndstoneBlueTopaz",
+    "5",
+	"oreEndstoneGarnetRed",
+    "5",
+	"oreEndstoneGarnetYellow",
+    "5",
+	"oreEndstoneJade",
+    "5",
+	"oreEndstoneVinteum",
+    "5",
+	"oreEndstoneLapis",
+    "5",
+	"oreEndstoneLazurite",
+    "5",
+	"oreEndstoneMysteriousCrystal",
+    "5",
+	"oreEndstoneInfusedFire",
+    "5",
+	"oreEndstoneInfusedAir",
+    "5",
+	"oreEndstoneInfusedWater",
+    "5",
+	"oreEndstoneInfusedEarth",
+    "5",
+	"oreEndstoneInfusedOrder",
+    "5",
+	"oreEndstoneInfusedEntropy",
+    "5",
+	"oreEndstoneNetherStar",
+    "5"
+  ],
+  "radius": 32,
+  "cost": 1,
```

Comment:
Questionable cost

Reply:
This was answered in Discord but basically, this is under the assumption that the people will likely have the Armok blood orb, trivializing any LP cost that is under 1 billion or 100 million, I forgot how much it was since. Although, now that I think about it, it's possible someone could rush towards this (granted they'd be in ~UHV tier and thus right next to infinity) and be granted this without the worry of LP costs, though it seems slim